sql/src/test/java/org/apache/druid/sql/calcite/parser/DruidSqlUnparseTest.java

```diff
@@ -41,7 +41,7 @@ public void testUnparseInsert() throws ParseException
     String sqlQuery = "INSERT INTO dst SELECT * FROM foo PARTITIONED BY ALL TIME";
     String prettySqlQuery = "INSERT INTO \"dst\"\n"
                      + "SELECT *\n"
-                     + "    FROM \"foo\" PARTITIONED BY ALL TIME";
+                     + "    FROM \"foo\" PARTITIONED BY ALL";
```

Comment:
I suppose this change is fine as `ALL` and `ALL TIME` are effectively the same? Or should we also explicitly handle `ALL TIME` to retain the query as-is?

Reply:
effectively the same. When the query is unparsed and rewritten, it writes the sqlNode for partitionedBy as a Granularity, which only has en eum for the equivalent effective granularity, `ALL`.